index.js

```diff
@@ -407,6 +407,21 @@ module.exports = {
 			},
 		],
 		'max-depth': 'warn',
+		'max-len': [
+			'warn',
+			{
+				code: 200,
+				ignoreComments: true,
+				ignoreUrls: true
+			},
+		],
+		'max-lines': [
+			'warn',
+			{
+				max: 2000,
+				skipComments: true,
```

Comment:
`max-lines` also has a way to exclude comments. I think we can lower the counter to 1500 thanks to this.

```suggestion
		'max-lines': [
			'warn',
			{
				max: 1500,
				skipComments: true,
```